py/server/tests/test_udf_numpy_args.py

```diff
@@ -392,6 +392,41 @@ def f31(p1: Optional[np.bool_], p2=None) -> bool:
             t2 = t.update(["X1 = f31(null, Y)"])
             self.assertEqual(10, t2.to_string("X1").count("true"))
 
+    def test_non_np_typehints(self):
+        py_types = {"int", "float"}
+
+        for p_type in py_types:
+            with self.subTest(p_type):
+                func_str = f"""
+def f(x: {p_type}) -> bool:  # note typing
+    return type(x) == {p_type}
+"""
+                exec(func_str, globals())
+                t = empty_table(1).update(["X = i", f"Y = f(({p_type})X)"])
```

Comment:
Seems like this should also be testing with `long` and `double` since the python types are wide.

Reply:
These tests are just to make sure that the converted args' types are what we expected, the actual values are not a concern here and they are tested in other test cases in the same test class.